centreon-archive/centreon-archived
# feat(Resource-status/Filter): store expand parameter in url and storage & fix multi-select criterias display issues

## Description

This allows the expand parameter to be stored in the URL query parameters & local storage, and fixes some display issues in the Criteria multi select spotted by @Thebarda 

## Type of change


## Target serie
- [x] 21.04.x (master)

## How to test

- expand the criterias
- Make sure the value is changed when changing the page and coming back
- display the criteria multi select and make sure the order stays the same when unselecting / selecting criterias
